website/docs/utilities/dialog-primitive/partials/guidelines/guidelines.md

```diff
@@ -0,0 +1,23 @@
+## Usage
+
+The DialogPrimitive sub-components are not intended to be used independently of each other or in isolation. If you are looking to create your own dialog-based component, use the `DialogPrimitive::Wrapper` as your foundation.
+
+A common example of this is a custom dialog-based element such as a SplitWindow/Pane/Panel which allows the user to interact with both the parent page and the content within the secondary element (often referred to as a “non-modal” dialog).
+
+!!! Info
+
+If you discover a use case outside of this example, please [contact](/about/support) the HDS team for assistance.
+!!!
+
+![Example of a non-modal example](/assets/components/dialog-primitives/dialog-primitive-non-modal-example.png)
+
+### Use within HDS components
+
+Examples of how to use the DialogPrimitive components can be found in the HDS [Modal](/components/modal) and [Flyout](/components/flyout) components which are composed using the DialogPrimitive. They are both types of dialogs, but have different use cases depending on the context, type of information they convey, and intended speed of interaction.
+
```

Comment:
@jorytindall I have doubts about this part:

"and intended speed of interaction."

Reply:
I reference interaction "speed" in the Flyout docs, but probably makes sense to remove it here since this a bit more of a generic/broad approach.